cmd/src/api.go

```diff
@@ -123,8 +127,25 @@ func gqlURL(endpoint string) string {
 	return endpoint + "/.api/graphql"
 }
 
+// authenticationHeader returns the header name and header value required to authenticate a request,
+// or `"", ""` if one calls this function when the config has no AccessToken information
+func authenticationHeader(cfg *config) (string, string) {
```

Comment:
This function is _always_ returning just the `Authorization` header, and returning it as a value makes the code significantly more complex.

I think that a better solution would be for this to only return the _value_ of the `Authorization ` header.